src/ser/mod.rs

```diff
@@ -1,18 +1,13 @@
 use std::error::Error as StdError;
+use std::result::Result as StdResult;
 use std::fmt::{Display, Formatter, Result as FmtResult};
 
 use serde::ser::{self, Serialize};
 
+#[deprecated]
```

Comment:
I wouldn't deprecate it. Better just move the new function to that module. If you really want to, we should give an explanation.

Reply:
I don't see why we need `pretty` module, it's empty and serves no solid purpose.

Reply:
And what `deprecated` tells you is basically that it's going to be cut off in the next breaking release, which is what I wanted to do in the first place :)

Reply:
Yes, but let's add an explanation that one should simply use `to_string_pretty` with a default config instead.

Reply:
So #[deprecated(since="0.1.4", note="please use `to_string_pretty` with `PrettyConfig::default()` instead")]